src/components/AdminOrderItem/index.tsx

```diff
@@ -1,46 +1,66 @@
-import React from 'react';
-import { Avatar, Checkbox, Row, Col } from 'antd';
+import React, { useState } from 'react';
+import { Avatar, Checkbox, Form, Input, Row, Col, Modal } from 'antd';
 import Icon from '@ant-design/icons';
+import { OrderItem } from '../../types/merch';
 import { ReactComponent as CommentBoxEmpty } from '../../assets/icons/CommentBoxEmpty.svg';
+import { ReactComponent as CommentBox } from '../../assets/icons/CommentBox.svg';
 
 import './style.less';
 
 interface AdminOrderItemProps {
-  orderItem: {
-    uuid: string;
-    itemName: string;
-    quantity: number;
-    fulfilled: boolean;
-    price: number;
-    description: string;
-    notes: string;
-  };
-  triggerModal: Function;
+  orderItem: OrderItem;
   setFulfill: Function;
+  setNote: Function;
 }
 
 const AdminOrderItem: React.FC<AdminOrderItemProps> = (props) => {
-  const { orderItem, triggerModal, setFulfill } = props;
+  const { orderItem, setFulfill, setNote } = props;
+  const [noteVisible, setNoteVisible] = useState(false);
+  const [scratchNote, setScratchNote] = useState(orderItem.notes || '');
+  const [checkboxDisabled, setCheckboxDisable] = useState(orderItem.fulfilled);
```

Comment:
Why not lift some of these local states to the container?

Reply:
I'm not quite exactly sure if I can lift `noteVisible` and `scratchNote`, since they are unique for each OrderItem. I'll try to deal with the `checkboxDisable`, though.

Reply:
Resolved in 48726aa.